url.bs

```diff
@@ -2247,6 +2247,18 @@ and then runs these steps:
      <li><p>If <a>c</a> is an <a>ASCII alphanumeric</a>, U+002B (+), U+002D (-), or U+002E (.),
      append <a>c</a>, <a lt="ASCII lowercase">lowercased</a>, to <var>buffer</var>.
 
+     <li>
+      <p>Otherwise, if <a>c</a> is U+003A (:) and <var>buffer</var>'s <a for=string>length</a> is 1 and 
+      <var>buffer</var> contains only an <a>ASCII alpha</a> and <a>remaining</a> starts with U+005C (\), then:
+
+      <ol>
+       <li><p>Set <var>url</var>'s <a for=url>scheme</a> to "<code>file</code>".
+       <li><p>Set <var>buffer</var> to the empty string.
+       <li><p>Replace every U+005C (\) code point in <a>remaining</a> with U+002F (/).
+       <li><p>Prepend "<code>///</code>" to <a>remaining</a>.
+       <li><p>Set <var>state</var> to <a>file state</a>.
```

Comment:
Is there a difference between this and immediately jumping to the path state?

Reply:
thank you yes this is simpler and cleaner, this is how I update